src/frontend/web_application/src/scenes/Contact/presenter.jsx

```diff
@@ -34,19 +50,65 @@ class Contact extends Component {
     this.props.updateContact({ contact, original });
   }
 
+  handleContactDelete({ contact }) {
+    this.props.removeContact({ contact });
+  }
+
+  handleClickTags() {
+    this.setState({ isTagModalOpen: true });
+  }
+
+  handleCloseTagsModal() {
+    this.setState({ isTagModalOpen: false });
+  }
+
+  renderTagsModal() {
+    const { contact, updateContact, __ } = this.props;
+    const count = contact.tags ? contact.tags.length : 0;
+    const title = [
```

Comment:
I think this works perfectly but I do not like the `key="1"` to prevent warning/error about array of nodes.

what about something more straight like:

```jsx
(<span>__('tags.header.title') <span className="m-tags-form__count">{__('tags.header.count', { count }) }</span></span>)
```